WordPress/twentytwentyfive
# Default single post template: Move the category below the title

**Description**
This PR updates the default template for the single post, and closes https://github.com/WordPress/twentytwentyfive/issues/291.

It moves the category below the featured image, to the same line as "Written by [authorname]",
because positioning it before the title is not as accessible as positioning it below the title.

The new text is: `Written by Author name in category, category 2, category 2`.
The text that was already in the template was not translation ready, so I created a new hidden pattern with the slug
`hidden-written-by`.

This text has two text colors: It uses `primary` for the plain text, and inherits the default link color, `contrast`,  for the links.
There is one inconsistency in this PR compared to the proposed design in https://github.com/WordPress/twentytwentyfive/issues/291, and that is that the category links are not underlined by default.
And that is because the underline is hidden by default in theme.json. Changing this would create _other_ miss-matches between the templates and the design elsewhere.

The word "in" is a paragraph. This can be changed to be the prefix of the terms block if that is preferred.

**Screenshots**
Editor:

<img width="1508" alt="The Written by text and the category on the single post template in the Site Editor" src="https://github.com/user-attachments/assets/0b93a90b-669d-4fcb-8eeb-faf60235c1ed">

Front, only the post meta:
<img width="365" alt="single-post-front--post-meta" src="https://github.com/user-attachments/assets/af559fed-aa22-483e-978e-5b52fb6ef641">



**Testing Instructions**
View the default single template in the editor and front and confirm if it matches the design.
